DSSE/environment/env_base.py

```diff
@@ -167,11 +166,11 @@ def is_valid_position(self, position: tuple[int, int]) -> bool:
     def default_drones_positions(self):
         counter_x = 0
         counter_y = 0
-        for agent in self.agents:
+        for agent_index in range(len(self.agents)):
```

Comment:
Maybe use enumerate instead of `range(len())`

Reply:
We only need the index on this loop, i don't see a reason.

Reply:
Fair enough 😂